test cases/unit/69 test env value/meson.build

```diff
@@ -0,0 +1,10 @@
+project('test env value')
+
+testpy = find_program('test.py')
+
+val = ['1', '2', '3']
+foreach v: val
+    test('check env', testpy, args: [v], env: {'TEST_VAR': v})
+endforeach
```

Comment:
The foreach here does not test anything. It's an artifact of when your test was updating an environment object.

Reply:
well, it tests different env values, we can remove it, but I would rather keep it.